### PR TITLE
Issue/s2e 200 websvc

### PR DESCRIPTION
**Pull request by vitalych**

### CODE REVIEW

File: s2e_env/commands/run.py [hunk 1]
```diff
@@ -202,10 +203,8 @@ def handle(self, *args, **options):
 
         self._start_time = datetime.datetime.now()
 
-        if 'cgc' not in self._project_desc['image']['os_name']:
-            self.warn('Real time statistics only supported for CGC binaries for now.')
-            self.warn('The UI will be empty.')
-            time.sleep(3)
+        if 'cgc' in self._project_desc['image']['os_name']:
+            self._cgc = True
```
Comment (adrianherrera) [inline s2e_env/commands/run.py]:
```python
self._cgc = 'cgc' in self._project_desc['image']['os_name']
```

File: s2e_env/server/coverage.py [hunk 2]
```diff
@@ -174,6 +174,3 @@ def get_tb_coverage(self):
 
     def get_summary(self):
```
Comment (adrianherrera) [inline s2e_env/server/coverage.py]: Can you please replace these getters with `@property` dectorators (see https://docs.python.org/2/library/functions.html#property)

E.g. `get_summary` becomes

```python
@property
def summary(self):
    return self._summary
```

Usage then becomes

```python
CollectorThreads.coverage.summary
```

File: s2e_env/commands/run.py [hunk 3]
```diff
@@ -283,6 +282,7 @@ def _get_data(self):
             binaries = "Waiting for analysis to start..."
 
         gs = CollectorThreads.stats.get_global_stats()
```
Comment (adrianherrera) [inline s2e_env/commands/run.py]: Replace with properties

**Comment by adrianherrera** [comment]
Looks good! Just a few minor style things to fix

File: s2e_env/templates/s2e-config.common.lua [hunk 4]
```diff
@@ -0,0 +1,98 @@
+function table.contains(table, element)
```
Comment (adrianherrera) [inline s2e_env/templates/s2e-config.common.lua]: I still don't like mixing code with configuration like this. Is it really necessary here?

Comment (vitalych) [inline s2e_env/templates/s2e-config.common.lua]: The thing is that we may have different features that require overlapping sets of plugins. I found it more readable to add plugins together with the feature definition. Unfortunately, that means that some plugins may be added twice to the table. So I had to create a function that checks first before adding.

**Pull request merged**